src/main/java/org/opensearch/security/configuration/ClusterInfoHolder.java

```diff
@@ -106,19 +72,4 @@ public Boolean hasNode(DiscoveryNode node) {
         
         return nodes.nodeExists(node)?Boolean.TRUE:Boolean.FALSE;
     }
-
-    private static boolean clusterHas6xNodes(ClusterState state) {
```

Comment:
Are we officially dropping support for migration from ES 6.x to OpenSearch versions in 3.0?

Reply:
What do you think about creating a separate issue to unblock this pull request so we can get main building again?

Off the cuff, I thought we only supported a migration like ES 7.8 -> OS 1.X -> OS 2.X -> OS 3.X.

Reply:
Who can confirm the migration path?

The documentation I have found mostly references upgrading from ES 7 -> OS 1

- https://opensearch.org/blog/technical-posts/2021/10/moving-from-opensource-elasticsearch-to-opensearch/
- https://opensearch.org/docs/latest/upgrade-to/upgrade-to/#upgrade-paths (Says Upgrade to OpenSearch, but doesn't specify a major version)